packages/ketcher-react/src/script/ui/state/server/index.js

```diff
@@ -132,6 +132,7 @@ export function analyse() {
       .catch((e) => {
         KetcherLogger.error('index.js::analyse', e);
         editor.errorHandler(e);
+        throw e;
```

Comment:
Why dont we implement the same way as `ketcher.eventBus.emit(GLOBAL_ERROR_HANDLER, error.message);` here and remove the old `editor.errorHandler(e)`?